tests/test_parser.py

```diff
@@ -159,20 +198,46 @@ def assert_coffee_stout_recipe(self, recipes):
         assert recipe.mash.steps[0].step_time == 60
         assert recipe.mash.steps[0].step_temp == 66.66666667
 
-        assert round(recipe.og, 4) == 1.0594
-        assert round(recipe.og_plato, 4) == 14.6092
-        assert round(recipe.fg, 4) == 1.0184
-        assert round(recipe.fg_plato, 4) == 4.684
-        assert floor(recipe.ibu) == 25
-        assert round(recipe.abv, 2) == 5.35
+        # should have mashing base information
+        assert(type(recipe.mash) is Mash)
```

Comment:
I am pretty sure PyLint will not like this. Probably:
```suggestion
        assert isinstance(recipe.mash, Mash)
```

Reply:
Change it. I also executed pylint against the tests folder and it showing some issues unrelated to the PR. It kept it as-is because I don't know how you want to deal with those things.

Updated the readme so that the commands include the tests folder.